tests/integration/cmd_push_test.go

```diff
@@ -286,7 +288,7 @@ var _ = Describe("odo push command tests", func() {
 				cmpName,
 				appName,
 				project,
-				[]string{"stat", filepath.Join(dir, "src", "server.js")},
+				[]string{"stat", strings.TrimSuffix(dir, "/") + "/src/server.js"}, // this doesn't use filepath.Join for a reason as this commands needs to run on linux, but the host machine can be windows
```

Comment:
Will ToSlash() work after the join operation?

Reply:
Didn’t get u? can u elaborate?

Reply:
It won’t because the host system would still be Windows which would give different slash the pod os - being linux

Reply:
> It won’t because the host system would still be Windows which would give different slash the pod os - being linux

`ToSlash` always uses `/` even on Windows

Reply:
you could use `filepath.ToSlash(filepath.Join(.....))` this should be much safer

Reply:
Yeah we are already using ToSlash() in our code base for converting all paths to linux based paths https://github.com/openshift/odo/blob/7c0fef571f2ee1b0e3ad8927759b3efbb14f8616/pkg/sync/sync.go#L31

Reply:
done